manifests/server/install.pp

```diff
@@ -31,6 +31,17 @@
     }
   }
 
+  # Prevent the master service running and preventing Apache from binding to the port
+  if $::puppet::server::passenger and $::osfamily == 'Debian' {
+    file { '/etc/default/puppetmaster':
```

Comment:
Sorry, I just noticed that I specified the relationship back to front, which may have confused matters. It now specifies that the config is _before_ the package.